volodymyrsamotiy/sonic-buildimage
# [Mellanox] [201911] Update SAI submodule pointer

* Set monitoring VLAN hostif up dy default (for VNET ping tool)

Signed-off-by: Volodymyr Samotiy <volodymyrs@nvidia.com>



#### Why I did it
To include below changes:
* Set monitoring VLAN hostif up dy default (for VNET ping tool)

#### How I did it
Updated SAI submodule pointer

#### How to verify it
1. Create VLAN hostif according to changes in PR: https://github.com/Azure/sonic-swss/pull/1645
2. Verify it is admin up by default

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

[Mellanox] [201911] Update SAI submodule pointer

#### A picture of a cute animal (not mandatory but encouraged)
